assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -5592,7 +5592,16 @@ public void doSave_toggle_remove_review(RunData data) {
         if (state.getAttribute(PEER_ASSESSMENT_ASSESSOR_ID) != null) {
             String peerAssessor = (String) state.getAttribute(PEER_ASSESSMENT_ASSESSOR_ID);
             ParameterParser params = data.getParameters();
-            String submissionId = params.getString("submissionId");
+            String submissionRef = params.getString("submissionId");
+            String submissionId = null;
+            if(submissionRef != null){
+                int i = submissionRef.lastIndexOf(Entity.SEPARATOR);
```

Comment:
hmmm so is there a "/" on the end of the id? could use:
```
submissionId = submissionId.endsWith("/") ? StringUtils.chop(submissionId) : submissionId;
```
But it would be nice to correct where its actually being added.

Reply:
An example: 
submissionRef is: /assignment/s/5574_P_2017_N_N/d2fc796a-096a-4ffb-8c0b-a53756c32e3b/20b6576e-7206-4553-8349-83f2b5b014c7

We only need the last part: 20b6576e-7206-4553-8349-83f2b5b014c7 to call

- assignmentPeerAssessmentService.getPeerAssessmentItem(submissionId, peerAssessor); 
- assignmentPeerAssessmentService.updateScore(submissionId, peerAssessor);
- integrateGradebook(state, aReference, associateGradebookAssignment, null, null, null, -1, null, submissionId, "update", -1); if is needed